src/Data/List/Relation/Binary/Permutation/Inductive/Properties.agda

```diff
@@ -233,6 +233,15 @@ module _ {a} {A : Set a} where
     { isCommutativeMonoid = ++-isCommutativeMonoid
     }
 
+  -- Another useful lemma
+
+  shifts : ∀ xs ys {zs : List A} → xs ++ ys ++ zs ↭ ys ++ xs ++ zs
+  shifts xs ys = begin
```

Comment:
Could you add the implicit parameter {zs} in explicitly and use it in the reasoning? Makes it much less mysterious what the underscores are and doesn't add much to the length.

Reply:
Yes, done